intern_program/intern_program_fall_2018_projects.md

```diff
@@ -34,3 +34,14 @@ This includes building a `.json` catalog of the existing datasets, integration a
 
 #### Skills Required
 Python programming, basic knowledge of html
+
+### Modeling time in the IRI Data Library (DL): new functionality needed
+
+#### Supervisor
+Rémi COUSIN, Climate Information Analyst
```

Comment:
Did you mean to capitalize every letter of your last name? Will the accent render correctly in HTML, or would it be better to use an HTML code?

Reply:
It's ok to drop the accent. Sorry I tend to capitalize last name in full... working a lot with international folks, it's sometimes not obvious which is first and which is last name...

Reply:
There will be no problem keeping the accent. We just may need to adjust after we push and see if it renders properly. I can fix it later if necessary no problem. Let's just go with normal capitalization of your last name.

Reply:
IRI and Ingrid are a very important part of the data science landscape at Lamont. I believe we should accept all the projects at this stage and then let the candidates determine which are best suited to their skills and interests.

Reply:
Agreed.